vm/systemSmartContracts/esdt.go

```diff
@@ -191,7 +191,7 @@ func (e *esdt) issue(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 		e.eei.AddReturnMessage(err.Error())
 		return vmcommon.UserError
 	}
-	if len(args.Arguments[0]) < minLengthForTickerName ||
+	if len(args.Arguments[0]) < int(esdtConfig.MinTokenNameLength) ||
```

Comment:
do we still use the const/var `minLengthForTickerName` ?

Reply:
It is used for ticker name